### PR TITLE
[DBC-203][Refactor] Apply new sign in use case

### PR DESCRIPTION
**Pull request by vantam87**
## Description
We have changed the sign-in logic that we use Torus service instead of our server.
- Remove the old signup logic and relevant
- Apply like new sign-in flow

## Note
- Remove login by email totally, we'll trigger login to Torus by google or apple instead.
- This PR has just cleaned up the code before implementing the get wallet use case then.

### CODE REVIEW

File: Wallet/Sources/Data/Repositories/Remote/TorusRepository.swift [hunk 1]
```diff
@@ -18,7 +17,7 @@ struct TorusRepository: TorusRepositoryProtocol {
         self.torus = torus
     }
 
-    func login(_ loginProviders: LoginProviders) -> Single<TorusModel> {
-        torus.triggerLogin(loginProviders)
+    func signIn(_ method: SigInMethod) -> Single<TorusModel> {
```
Comment (spica) [inline Wallet/Sources/Data/Repositories/Remote/TorusRepository.swift:20]: SigInMethod -> SignInMethod

File: Wallet/Sources/Models/SignInModel.swift [hunk 2]
```diff
@@ -6,6 +6,10 @@
 //  Copyright © 2021 MVL. All rights reserved.
 //
 
+enum SigInMethod {
+    case google, apple
```
Comment (spica) [inline Wallet/Sources/Models/SignInModel.swift:10]: So this google / apple login also will be removed eventually?

Comment (vantam87) [reply Wallet/Sources/Models/SignInModel.swift:10]: No, it doesn't. We'll sign in by google and apple

Comment (spica) [reply Wallet/Sources/Models/SignInModel.swift:10]: ah ok

**Review by spica — APPROVED** [review]

**Pull request merged**